### ISSUE TITLE
More VAD options & flexibility (suppress VAD, custom segments)

### ISSUE THREAD

**Issue by exactstat** [m-bain/whisperX#338]
REQUEST:

1. add a vad_segments parameter to the .transcribe() method (and don't use internal VAD in case of external segments use)
2. add an option to disable VAD

REASON:

1. I want to use my custom vad_segments. 
2. Also, I want to use WhisperX without a VAD (for single audio)

Thanks!

**Comment by pprobst** [comment]
I agree! In my use case, I already have multiple < 30s audio segments and I'd like the option to skip VAD when transcribing.
